modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryRelatedObjectsResourceTest.java

```diff
@@ -634,6 +641,174 @@ public void testDeleteCustomObjectEntry() throws Exception {
+			objectEntry1JSONObject.get("externalReferenceCode"));
+
+		JSONArray objectEntry2JSONArray = objectEntry1JSONObject.getJSONArray(
+			objectRelationship1.getName());
+
+		Assert.assertNotNull(objectEntry2JSONArray);
+
+		JSONObject objectEntry2JSONObject = objectEntry2JSONArray.getJSONObject(
+			0);
+
+		Assert.assertEquals(
+			_objectEntry2.getExternalReferenceCode(),
+			objectEntry2JSONObject.get("externalReferenceCode"));
+	}
+
+	@Test
+	@TestInfo("LPD-60419")
+	public void testExportAndImportRelatedObjectEntryWithDifferentScopeAcrossSites()
+		throws Exception {
+
+		ObjectDefinition siteScopedObjectDefinition =
+			ObjectDefinitionTestUtil.publishObjectDefinition(
+				Collections.singletonList(
+					ObjectFieldUtil.createObjectField(
+						ObjectFieldConstants.BUSINESS_TYPE_TEXT,
+						ObjectFieldConstants.DB_TYPE_STRING, true, true, null,
+						RandomTestUtil.randomString(), _OBJECT_FIELD_NAME_1,
+						false)),
+				ObjectDefinitionConstants.SCOPE_SITE);
+
+		_objectDefinitions.add(siteScopedObjectDefinition);
+
+		ObjectEntry objectEntry2 = ObjectEntryTestUtil.addObjectEntry(
+			siteScopedObjectDefinition, _OBJECT_FIELD_NAME_1,
+			_OBJECT_FIELD_VALUE_2);
+
+		Group group = GroupTestUtil.addGroup();
+
+		ObjectEntry objectEntry3 = ObjectEntryTestUtil.addObjectEntry(
+			group.getGroupId(), siteScopedObjectDefinition,
+			HashMapBuilder.<String, Serializable>put(
+				_OBJECT_FIELD_NAME_1, _OBJECT_FIELD_VALUE_2
+			).build());
+
+		ObjectRelationship objectRelationship = _addObjectRelationship(
+			_objectDefinition1, siteScopedObjectDefinition,
+			_objectEntry1.getPrimaryKey(), objectEntry2.getPrimaryKey(),
+			ObjectRelationshipConstants.TYPE_MANY_TO_MANY);
+
+		ObjectRelationshipTestUtil.relateObjectEntries(
+			_objectEntry1.getPrimaryKey(), objectEntry3.getPrimaryKey(),
+			objectRelationship, TestPropsValues.getUserId());
+
+		JSONObject objectEntryExportJSONObject =
+			HTTPTestUtil.invokeToJSONObject(
+				null,
+				_getEndpoint(
+					String.valueOf(objectEntry2.getObjectEntryId()),
+					objectRelationship, siteScopedObjectDefinition),
+				Http.Method.GET);
+
+		objectEntryExportJSONObject.remove("actions");
+
+		JSONObject importedObjectEntryJSONObject =
+			HTTPTestUtil.invokeToJSONObject(
+				objectEntryExportJSONObject.toString(),
+				_getSiteScopedObjectDefinitionEndpoint(
+					siteScopedObjectDefinition, group.getGroupId()),
+				Http.Method.POST);
+
+		Assert.assertEquals(
+			0,
+			importedObjectEntryJSONObject.getJSONObject(
+				"status"
+			).get(
+				"code"
+			));
+
+		JSONObject objectEntriesPageJSONObject =
+			HTTPTestUtil.invokeToJSONObject(
+				null, _getEndpoint(objectRelationship.getName()),
+				Http.Method.GET);
+
+		JSONArray siteScopedObjectEntriesJSONArray =
+			objectEntriesPageJSONObject.getJSONArray("items");
+
+		Assert.assertNotNull(siteScopedObjectEntriesJSONArray);
+		Assert.assertEquals(3, siteScopedObjectEntriesJSONArray.length());
```

Comment:
Let’s replace that custom JSON assertion with the pattern from testGetRelatedObjectEntryWithDifferentScope. To fetch multiple layers of nested fields, pass the nestedFieldsDepth parameter equal to the number of layers you want. For example:

```suggestion
		JSONAssert.assertEquals(
			JSONUtil.put(
				_OBJECT_FIELD_NAME_1, _OBJECT_FIELD_VALUE_2
			).put(
				"externalReferenceCode", objectEntry2.getExternalReferenceCode()
			).put(
				objectRelationship1.getName(),
				JSONUtil.putAll(
					JSONUtil.put(
						_OBJECT_FIELD_NAME_1, _OBJECT_FIELD_VALUE_1
					).put(
						"externalReferenceCode",
						_objectEntry1.getExternalReferenceCode()
					).put(
					objectRelationship2.getName(),
					JSONUtil.putAll(
						JSONUtil.put(
							_OBJECT_FIELD_NAME_1, _OBJECT_FIELD_VALUE_1
						).put(
							"externalReferenceCode",
							_objectEntry1.getExternalReferenceCode()
						))
				))
			).toString(),
			HTTPTestUtil.invokeToJSONObject(
				null,
				_getEndpoint(
					String.valueOf(objectEntry2.getPrimaryKey()),
					objectRelationship, siteScopedObjectDefinition),
				Http.Method.GET
			).toString(),
			JSONCompareMode.LENIENT);
```